sysid-application/src/main/native/cpp/generation/ConfigManager.cpp

```diff
@@ -38,6 +41,13 @@ wpi::json ConfigManager::Generate(size_t occupied) {
   }
   json["motor controllers"] = motorControllers;
 
+  // Add CANivore busses
+  canivoreNames.clear();
+  for (size_t i = 0; i < occupied; i++) {
+    canivoreNames.push_back(std::string(m_config.canivoreNames[i].data()));
+  }
+  json["canivore names"] = canivoreNames;
+
```

Comment:
Don't forget that if you're going to add new keys to the JSON that you need to update the `ReadJSON` method to make sure that it's in the json keys array and that the config is updated when its read.

Reply:
Is the `is drivetrain` key intentionally omitted from the ReadJSON method? I noticed that's missing in my review of the method.